uiowa/uiowa
# [Hotfix] Slider JS aggregation

Not sure if this fixes this issue, but this is the only thing that shows up in the logs after testing. 

<img width="451" alt="Screenshot 2024-11-05 at 10 58 06 AM" src="https://github.com/user-attachments/assets/22bbca97-d485-4e7c-8207-2950a955e7e1">


# How to test
1. Enable local JS aggregation for dermatology.medicine.uiowa.edu
```
ddev blt ds --site=dermatology.medicine.uiowa.edu  && ddev drush @medicinedermatology.local uli  /node/1/layout
```
2. Add Slider to the page and save.  Edit the slider again and save.  Check logs for this error:

<img width="1057" alt="Screenshot 2024-11-05 at 2 00 30 PM" src="https://github.com/user-attachments/assets/d567f313-7348-4a5b-8f77-018cb795b645">
3. Change slider back to `preprocess: false` and repeat testing steps.
